### PR TITLE
Include tip on overcoming UDP protocol error with Vagrant & NFS

### PR DESCRIPTION
**Pull request by strzibny**

### CODE REVIEW

File: tools/vagrant/vagrant-nfs.md [hunk 1]
```diff
@@ -39,6 +39,20 @@ Vagrant.configure("2") do |config|
 end
 ```
 
+One of the common errors when using NFS this way might be that your NFS version does not support UDP protocol which gets added by Vagrant as a default. 
+
+The error says `mount.nfs: requested NFS version or transport protocol is not supported`.
```
Comment (pvalena) [inline tools/vagrant/vagrant-nfs.md]: How about rewording to something simillar to:
```
In that case the error message is ` ... `. You can work around this by disabling UDP with `nfs_udp` option: 
[ . . . ]
```

File: tools/vagrant/vagrant-nfs.md [hunk 2]
```diff
@@ -39,6 +39,20 @@ Vagrant.configure("2") do |config|
...
+
+In that case it's enough to disable UDP with `nfs_udp` option:
+
+```
+Vagrant.configure("2") do |config|
+  # ...
+
+  config.vm.synced_folder ".", "/vagrant", type: "nfs", nfs_udp: false
```
Comment (pvalena) [inline tools/vagrant/vagrant-nfs.md]: > config.vm.synced_folder ".", "/vagrant", type: "nfs", nfs_udp: false

Would not this line simply suffice? Is it worth repeating `Vagrant.configure("2") do |config|`?
WDYT?
￼

**Comment by strzibny** [comment]
I updated it as you suggested.

**Comment by pvalena** [comment]
Great, thanks!
